Eject function with README.md (#1925)
kyma-project/serverless: components/runtimes/nodejs22/README_template.md

# Ejected Functions Workspace

> This folder contains all most important block to test, run, deploy and productize your buisness application and consists of following elements:

Reviewer: ```suggestion
This folder contains the most important block to test, run, deploy, and productize your business application and consists of the following elements:
```

* `handler.js` - source code of the buisness application
* `server.mjs` - server source code needed to run the handler.js

> * `package.json` - dependencies file required for the handler.js and the server.mjs to run

Reviewer: ```suggestion
* `package.json` - dependencies file required for the `handler.js` and the `server.mjs` files to run
```

* `lib/` - directory containing server SDK (like cloudevents or tracing functionality)

> * `resources/` - directory with basic Kubernetes resources needed to deploy application on a cluster

Reviewer: ```suggestion
* `resources/` - directory with basic Kubernetes resources required to deploy the application on a cluster
```

* `Dockerfile` - application image definition
* `Makefile` - basic portion of automations and scripts

## Scripts and Automations

> The `Makefile` file is designed to speed up some processes like running application locally, build or deploy it on a k3d cluster.

Reviewer: ```suggestion
The `Makefile` file is designed to speed up processes such as running an application locally or building or deploying it on a k3d cluster.
```

Read more about possibilities and functionalites by running the `make help` target.

> ### To run application locally

Reviewer: ```suggestion
### Run Application Locally
```

> >NOTE: because application is run outside of the cluster it cannot simply reach in-cluster services and use container envs. It is strongly recommend to use following target to test application without such dependecies or mock them and export container envs manually.

Reviewer: ```suggestion
> [!NOTE] 
> Because the application is run outside the cluster, it cannot simply reach in-cluster services and use container envs. It is strongly recommended to use the following target to test the application without such dependencies or mock them and export container envs manually.
```

```bash
export FUNC_NAME=<name>
export FUNC_RUNTIME=<runtime>
export SERVICE_NAMESPACE=<SERVICE_NAMESPACE>

make run
```

> ### To deploy application on k3d

Reviewer: ```suggestion
### Deploy Application on k3d
```

> The workspace is designed to easly start working on productization of buisness application ejected from the Function and allows to test it on k3d cluster by building it, importing and deploying on it:

Reviewer: ```suggestion
The workspace is designed to easily start working on the productization of a business application extracted from Function. It allows testing it on a k3d cluster by building, importing, and deploying it:
```

```bash
make k3d-deploy
```

> ### To build and push application

Reviewer: ```suggestion
### Build and Push Application
```
